Service/src/routes/watchlist.py

```diff
@@ -9,6 +9,7 @@
 @watchlist_api.route('/api/watchlist', methods=['POST'])
 def post_watchlist():
    body = request.get_json()
+   g.log.info(str(body))
```

Comment:
use .debug when debugging

and this is also a blocker for merge because this log shouldn't be needed

Reply:
Removed